### ISSUE TITLE
Alternative to this library

### ISSUE THREAD

**Issue by aliehsan90** [StephenChou1017/react-big-scheduler#225]
This library is good however when there are alot of events, rendering becomes really slow. Has anyone tried an alternative to this library that is really fast with rendering 100s of events?

**Comment by Kennn-dev** [comment]
You can try Fullcalendar  , Syncfusion Scheduler , maybe can help this problem , but with pricing

**Comment by madzadev** [comment]
@aliehsan90 yup, see [#124](https://github.com/StephenChou1017/react-big-scheduler/issues/124) for more 😉

**Comment by hbatalhaStch** [comment]
https://github.com/hbatalhaStch/react-big-scheduler

see https://github.com/StephenChou1017/react-big-scheduler/issues/124#issuecomment-1400772300 for more info
